libs/core/src/lib/calendar/patch-deprecated-i18n-labels.ts

```diff
@@ -0,0 +1,29 @@
+import { inject } from '@angular/core';
+import { FD_LANGUAGE, FdLanguage } from '@fundamental-ngx/i18n';
+import { Observable, map, startWith, switchMap } from 'rxjs';
+import { CalendarI18nLabels } from './i18n/calendar-i18n-labels';
+
+/**
+ * Use deprecated `CalendarI18nLabels` to patch `FD_LANGUAGE` observable
+ */
+export function patchDeprecatedI18nLabels(): Observable<FdLanguage> {
+    const deprecatedSymbol = inject(CalendarI18nLabels, { optional: true });
+    if (deprecatedSymbol) {
```

Comment:
```suggestion
    if (deprecatedSymbol && deprecatedSymbol.labelsChange) {
```

I reverted the core calendar i18n example and the page would not load due to console errors. Need this check to allow backwards compatibility

Reply:
I will fix that, but differently. Thanks